main.tf

```diff
@@ -74,6 +74,7 @@ module "vpc" {
   vpc_id              = var.vpc_id
   region              = var.location
   security_group_id   = local.security_group_id
+  raw_sec_group_id    = var.security_group_id
```

Comment:
Discussed the requirement for this additional security group input for the aws_vpc module with Thomas. May consider refactoring in the future to use only var.security_group_id instead of also using the local.security_group_id as inputs but leaving as is for now.